libadcc/pyiface/export_Tensor.cc

```diff
@@ -346,6 +349,17 @@ static py::object Tensor___repr__(const Tensor& self) {
   return Tensor___str__(self);
 }
 
+static ten_ptr set_lt_scalar(const py::float_ n) {
+  auto adcmem_ptr = std::shared_ptr<AdcMemory>(new AdcMemory());
```

Comment:
I'm a bit worried about the following:

1. Creating a new, possibly dangling `AdcMemory` instance here. AFAIK, this is usually only created once and used throughout.
2. Importing `libtensor` directly in the pybind export file. Usually, we only import `libtensor` in `TensorImpl.cc`.

If you have a second, @mfherbst, what do you think about it?

If I understand correctly, this functionality just enables using a single number as item in an amplitude vector. With some boilerplate, this could possibly very easily achieved on the Python level, right?